package/firmware/ipq-wifi/Makefile

```diff
@@ -22,9 +22,13 @@ endef
 # You can find instructions and examples on the linux-wireless wiki:
 # <https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles>
 #
+# Use ath10k-bdencoder from https://github.com/qca/qca-swiss-army-knife.git
+# to generate the board-* files here.
+#
 
 ALLWIFIBOARDS:= \
 	avm_fritzrepeater-1200 \
+	dlink_dap2610 \
```

Comment:
Have you posted the boardfiles to the ath10k mailinglist as well? If not, then please follow the instructions in the comment above and post these to the mailing-list. Thanks.

Reply:
I have now B)